dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jdbcJavaEE-1.0.feature

```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jdbcJavaEE-1.0
+IBM-Provision-Capability:\
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.transaction-1.2)(osgi.identity=com.ibm.websphere.appserver.transaction-1.1)))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.jdbc-4.3)(osgi.identity=com.ibm.websphere.appserver.jdbc-4.2)(osgi.identity=com.ibm.websphere.appserver.jdbc-4.1)(osgi.identity=com.ibm.websphere.appserver.jdbc-4.0)))"
+IBM-Install-Policy: when-satisfied
+-bundles=com.ibm.ws.jdbc
+kind=ga
```

Comment:
noship please

Reply:
This is the Java EE autofeature. It would be a breaking change to make in nonship, in which case the existing jdbc features would become unusable for Java EE.  Hopefully you just misread this as being a Jakarta autofeature.  The change you requested cannot be made.

Reply:
OK, fair enough